test/functional/ui/statuscolumn_spec.lua

```diff
@@ -589,6 +590,29 @@ describe('statuscolumn', function()
         meths.input_mouse('left', 'press', '', 0, 7, 7)
         eq('0 1 l 11', eval("g:testvar"))
       end)
+
+      it('selecting popupmenu does not drag mouse', function()
+        screen:try_resize(screen._width, 2)
```

Comment:
I'd suggest using `screen:set_default_attr_ids()` to make the screen states look less confusing.